protostar-rust/tests/data/dispatchers/tests/using_dispatchers.cairo

```diff
@@ -0,0 +1,54 @@
+use array::ArrayTrait;
+use result::ResultTrait;
+use option::OptionTrait;
+use traits::TryInto;
+use starknet::ContractAddress;
+use starknet::Felt252TryIntoContractAddress;
+
+use dispatchers::erc20::IERC20Dispatcher;
+use dispatchers::erc20::IERC20DispatcherTrait;
+use dispatchers::hello_starknet::IHelloStarknetDispatcher;
+use dispatchers::hello_starknet::IHelloStarknetDispatcherTrait;
+
+
+#[test]
+fn call_and_invoke() {
+    let contract_address = deploy_contract('hello_starknet', @ArrayTrait::new()).unwrap();
+    let contract_address: ContractAddress = contract_address.try_into().unwrap();
+    let dispatcher = IHelloStarknetDispatcher { contract_address };
+
+    let balance = dispatcher.get_balance();
+    assert(balance == 0, 'balance == 0');
+
+    dispatcher.increase_balance(100);
+
+    let balance = dispatcher.get_balance();
+    assert(balance == 100, 'balance == 100');
+}
+
+#[test]
+fn advanced_types() {
+    let mut calldata = ArrayTrait::new();
+    calldata.append('token');   // name
+    calldata.append('TKN');     // symbol
+    calldata.append(18);        // decimals
+    calldata.append(0);         // initial supply high
+    calldata.append(1111);      // initial supply low
+    calldata.append(1234);      // recipient
+
+    let contract_address = deploy_contract('erc20', @calldata).unwrap();
```

Comment:
There is no way to call a constructor using dispatcher 🤔

Reply:
According to cairo book, it's not possible to even define it. 

> Interfaces in Cairo are traits with the #[abi] attribute. If you are new to [traits](https://cairo-book.github.io/ch07-02-traits-in-cairo.html), check out the dedicated chapter on traits.
> 
> For your Cairo code to qualify as an interface, it must meet the following requirements:
> 
> Must be appended with the #[abi] attribute.
> Your interface functions should have no implementations.
> You must explicitly declare the function's decorator.
> Your interface should not declare a constructor.
> Your interface should not declare state variables.

https://cairo-book.github.io/ch99-02-01-abis-and-interfaces.html#interface